arbnode/seq_coordinator.go

```diff
@@ -522,6 +594,10 @@ func (c *SeqCoordinator) update(ctx context.Context) time.Duration {
 		log.Error("cannot read message count", "err", err)
 		return c.config.UpdateInterval
 	}
+	remoteFinalizedMsgCount, err := c.getRemoteFinalizedMsgCount(ctx)
+	if err != nil {
+		log.Warn("Cannot get remote finalized message count, might encounter failed to read message warnings later", "err", err)
```

Comment:
Why only logging the error here instead of returning?

Reply:
the reason here is that finalizedMsgCount key is only written by the chosen sequencer. We want others to depend on their on l1 readers to catchup until finalized and preemptively not look for older messages, but in a case such as the chosen sequencer not being able to write finalizedMsgCount Key for the first time, we dont want the non-chosen sequencers to stop catching up via redis altogether.

Reply:
Makes sense :)